helper-cli/src/main/kotlin/utils/PathExcludeGenerator.kt

```diff
@@ -42,8 +42,10 @@ internal object PathExcludeGenerator {
         val dirsToExclude = mutableMapOf<File, PathExcludeReason>()
 
         dirs.forEach { dir ->
-            PATH_EXCLUDES_REASON_FOR_DIR_NAME[dir.name]?.let { reason ->
-                dirsToExclude += dir to reason
+            PATH_EXCLUDES_REASON_FOR_DIR_NAME.forEach { (dirname, reason) ->
```

Comment:
Commit message:
- Typo in "cleaar"
- What does the "just" in "Decided to use just wildcards" refer to? Why "just"? What's the alternative?